reactor-netty5-http/src/main/java/reactor/netty5/http/client/HttpClient.java

```diff
@@ -441,50 +439,13 @@ public final HttpClient cookie(Cookie cookie) {
 	/**
 	 * Apply cookies configuration.
 	 *
-	 * @param cookieBuilder the header {@link Consumer} to invoke before requesting
+	 * @param cookieBuilder the header {@link Supplier} to invoke before requesting
 	 *
 	 * @return a new {@link HttpClient}
 	 */
-	public final HttpClient cookie(String name, Consumer<? super Cookie> cookieBuilder) {
-		Objects.requireNonNull(name, "name");
+	public final HttpClient cookie(Supplier<HttpCookiePair> cookieBuilder) {
```

Comment:
This method is replacing the old _public final HttpClient cookie(String name, Consumer<? super Cookie> cookieBuilder)_ see https://github.com/reactor/reactor-netty/blob/netty5/reactor-netty5-http/src/main/java/reactor/netty5/http/client/HttpClient.java#L448

Since the new _HttpCookiePair_ is immutable, I have replaced the old method by the new proposed method: "_public final HttpClient cookie(Supplier<HttpCookiePair> cookieBuilder)_"

Please confirm if this method should be really removed or if the old method should be replaced by something else ?

Reply:
There is no need to keep this method. I just wanted to double check if the old _cookie(String name, Consumer<? super Cookie>_ method can be removed, or if we need to replace the old method using a HttpCookiePair lambda supplier (like _cookie(Supplier<HttpCookiePair> cookieBuilder)_).

I will remove the method.